docs/reference/ImageOps.rst

```diff
@@ -12,18 +12,48 @@ only work on L and RGB images.
 
 .. autofunction:: autocontrast
 .. autofunction:: colorize
-.. autofunction:: contain
-.. autofunction:: pad
 .. autofunction:: crop
 .. autofunction:: scale
 .. autofunction:: deform
 .. autofunction:: equalize
 .. autofunction:: expand
-.. autofunction:: fit
 .. autofunction:: flip
 .. autofunction:: grayscale
 .. autofunction:: invert
 .. autofunction:: mirror
 .. autofunction:: posterize
 .. autofunction:: solarize
 .. autofunction:: exif_transpose
+
+.. _relative-resize:
+
+Resize relative to a given size
+-------------------------------
+
+::
+
+    from PIL import Image, ImageOps
+    size = (100, 150)
+    with Image.open("Tests/images/hopper.png") as im:
+        ImageOps.contain(im, size).save("imageops_contain.png")
+        ImageOps.cover(im, size).save("imageops_cover.png")
+        ImageOps.fit(im, size).save("imageops_fit.png")
+        ImageOps.pad(im, size, color="#f00").save("imageops_pad.png")
+
+        # thumbnail() can also be used,
+        # but will modify the image object in place
+        im.thumbnail(size)
+        im.save("imageops_pad.png")
+
++------+-------------------------------------------+--------------------------------------------+------------------------------------------+----------------------------------------+----------------------------------------+
+|      | :py:meth:`~PIL.Image.Image.thumbnail`     | :meth:`contain`                            | :meth:`cover`                            | :meth:`fit`                            | :meth:`pad`                            |
++======+===========================================+============================================+==========================================+========================================+========================================+
+|Size  | (100, 100)                                | (100, 100)                                 | (150, 150)                               | (150, 100)                             | (150, 100)                             |
++------+-------------------------------------------+--------------------------------------------+------------------------------------------+----------------------------------------+----------------------------------------+
+|Image | .. image:: ../example/image_thumbnail.png | .. image:: ../example/imageops_contain.png | .. image:: ../example/imageops_cover.png | .. image:: ../example/imageops_fit.png | .. image:: ../example/imageops_pad.png |
++------+-------------------------------------------+--------------------------------------------+------------------------------------------+----------------------------------------+----------------------------------------+
```

Comment:
It's not clear that the numbers in the table are actual sizes, not requested.
I suggest to use `(150, 100)` notation for arguments and `100×100` notation for actual sizes.
For the same reason the actual sizes could be placed beneath the image.

```suggestion
|Requested size  | ``(150, 100)``   | ``(150, 100)``  | ``(150, 150)``  | ``(150, 100)``  | ``(150, 100)``  |
+----------------+------------------+-----------------+-----------------+-----------------+-----------------+
|Image | .. image:: ../example/image_thumbnail.png | .. image:: ../example/imageops_contain.png | .. image:: ../example/imageops_cover.png | .. image:: ../example/imageops_fit.png | .. image:: ../example/imageops_pad.png |
+----------------+------------------+-----------------+-----------------+-----------------+-----------------+
|Resulting size  | ``100×100``      | ``100×100``     | ``150×150``     | ``150×100``     | ``150×100``     |
```

Reply:
I've pushed a version of this - I chose 'Given size' rather than 'Requested size', because I think 'Requested size' makes it sound like that it is what the output should match.